### PR TITLE
Add image encoder / decoder to proposed solutions section.

### PR DESCRIPTION
**Pull request by jabolopes**
Hi Peter,

This fixes a few typos and adds image encoding / decoding to the proposed solutions section.

Thanks!

Jose

### CODE REVIEW

**Comment by jabolopes** [comment]
Hi Steve, I've updated the PR based on your feedback.

I believe ImageEncoder / ImageDecoder may be stateful because the codec may be configured with codec specific parameters, for example, ImageEncoder({codec: "jpeg", quality: 0.9}). It also makes the API more consistent, because Image is presented to developers just like Audio or Video. And we are also exploring ways of integrating WebAssembly with codecs, so the object may be backed by a WebAssembly program to run the actual codec.

What do you think?

File: explainer.md [hunk 1]
```diff
@@ -62,20 +62,24 @@ We build on top of [WHATWG Streams](https://streams.spec.whatwg.org/) (in partic
...
 A **VideoTrackReader** converts a MediaStreamTrack into a ReadableStream of DecodedVideoFrame.
 
-A **VideoEncoder** is a TransformStream from DecodeVideoFrame to EncodedVideoFrame.
+A **VideoEncoder** is a TransformStream from DecodedVideoFrame to EncodedVideoFrame.
 
 A **VideoDecoder** is a TransformStream from EncodedVideoFrame to DecodedVideoFrame.
 
 A **VideoTrackWriter** converts a WritableStream of DecodedVideoFrame into a MediaStreamTrack.
 
+An **ImageEncoder** encodes a single image from [ImageData](http://go/mdn/API/ImageData) to EncodedImageData.
```
[the remark this answers is not in the document]
Comment (jabolopes) [reply explainer.md]: Done.

**Comment by sandersdan** [comment]
I believe this request is out of date. ImageDecoder is currently being maintained externally (https://github.com/dalecurtis/image-decoder-api), but I do expect it will be merged here eventually.

**Comment by rektide** [comment]
> *I believe this request is out of date. ImageDecoder is currently being maintained externally*

There doesn't seem to be anything for `ImageEncoder`, which I would also like to see.

I did just open an issue https://github.com/dalecurtis/image-decoder-api/issues/7 . I'll try to follow up & find how we might pursue this. This would be so lovely to get.

**Comment by talltyler** [comment]
What is the status on ImageEncoder? The link above is now broken because issues has been disabled.
Having a ImageEncoder could solve many the needs of many apps creating gifs and animated pngs.

**Comment by dalecurtis** [comment]
https://github.com/w3c/webcodecs/issues/204 is the tracking issue, but no active plans for it at the moment since `toBlob` exists.

**Pull request closed without merging**